src/SGB_Command_Border.md

```diff
@@ -70,16 +71,21 @@ window to be displayed inside. Non-transparent border data will cover
 the Game Boy window (for example, *Mario's Picross* does this, as does
 *WildSnake* to a lesser extent).
 
+A border designed for a modern (post-2006) widescreen television may use the center 256x176 pixels and leave the top and bottom 24 lines blank.
+Using letterbox allows more tile variety in the portion of the border that a widescreen TV's zoom mode does not cut off.
+
 All borders repeat tiles. Assuming that the blank space for the GB
-screen is a single tile, as is the letterbox in a widescreen border, a
+screen is a blank tile, and the letterbox (if any) is a solid tile, a
 border defining all unique tiles would have to define this many tiles:
 
--   (256\*224-160\*144)/64+1 = 537 tiles in fullscreen border
--   (256\*176-160\*144)/64+2 = 346 tiles in widescreen border
+-   (256\*224-160\*144)/64+1 = 537 tiles in full-screen border
+-   (256\*176-160\*144)/64+2 = 346 tiles in letterboxed border
```

Comment:
It would be nice if this could use the same math rendering as #350 (polyfills & all)...

https://github.com/gbdev/pandocs/blob/a4049ecdb0fdfe63a8478e012511cf6b49198014/src/Audio_details.md#L169

...but maybe we can tackle that later.

Reply:
Yes, let's open an Issue about this and merge this as it is